src/cogs/fun.py

```diff
@@ -0,0 +1,143 @@
+import random
+
+import aiohttp
+import discord
+import requests as requests
+from discord.ext import commands
+from datetime import datetime
+
+
+class Fun(commands.Cog):
+    """Fun Bot Commands"""
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name='8ball',
+                      description="Answers a yes/no question.",
+                      brief="Answers from the beyond.",
+                      aliases=['eight_ball', 'eightball', '8-ball'],
+                      pass_context=True)
+    async def eight_ball(self, ctx):
+        possible_responses = [
+            'That is a resounding no',
+            'It is not looking likely',
+            'Too hard to tell',
+            'It is quite possible',
+            'Definitely',
+            'Why don\'t you figure it out yourself you lazy motherfucker',
+            'Yes, but do it drunk as fuck!',
+            'My sources say no, but they also said Hillary would win.',
+            'Do not swipe right, it\'s your cousin',
+            'Do swipe right, it\'s your cousin!',
+            'Do what Jesus would do, die at the age of 33',
+            'Trump uses me when deciding to go to war'
+
+        ]
+        em = discord.Embed(color=self.bot.config['colors']['default'])
+        em.title = ":mage: The 8 ball wizard says..."
+        em.description = (random.choice(possible_responses) + ", " + ctx.message.author.mention)
+        await ctx.send(embed=em)
+
+    @commands.command(name="covid")
+    async def covid(self, ctx, *, country=None):
+        """Updated information about Covid-19."""
+        try:
+            if country is None:
+                embed = discord.Embed(title="This command is used like this: ```?covid [country]```", colour=0xff0000,
```

Comment:
Why is "country" optional if we are going to send an error message when it's not included?